os/arch/arm/src/imxrt/imxrt_flash.h

```diff
@@ -136,6 +136,7 @@ extern "C" {
 #define EXTERN extern
 #endif
 
+#if !defined(CONFIG_ARCH_CHIP_FAMILY_IMXRT102x)
```

Comment:
Why did you add this..? This function seems to be used only for imxrt1020 board...